packages/account/src/Components/currency-selector/radio-button.tsx

```diff
@@ -1,60 +1,32 @@
-import React, { InputHTMLAttributes, AllHTMLAttributes, ReactElement } from 'react';
+import React, { AllHTMLAttributes } from 'react';
```

Comment:
Usually we don’t extract Types from react separately, can we use them as usual React.AllHTMLAttributes?

Reply:
React. AllHTMLAttributes shows a TS error for props.status :(